MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeTests.m

```diff
@@ -800,4 +796,29 @@ - (void)testDismissEmbeddedSafariWhenDisabling {
                              }];
 }
 
+- (void)testShowDistributeDisabledAlert {
```

Comment:
You may also wants some tests to make sure the alert is displayed when module is disabled instead of the usual choices and  that it's not displayed if module is enabled